android_jni/avifandroidjni/src/main/java/org/aomedia/avif/android/AvifDecoder.java

```diff
@@ -110,10 +110,11 @@ public static boolean decode(ByteBuffer encoded, int length, Bitmap bitmap) {
    * @param bitmap The decoded pixels will be copied into the bitmap.
    *     If the bitmap dimensions do not match the decoded image's dimensions,
    *               then the decoded image will be scaled to match the bitmap's dimensions.
-   * @param threads Number of threads to be used for the AVIF decode. Zero means use number of CPU
-   *     cores as the thread count. Negative values are invalid. When this value is > 0, it is
-   *     simply mapped to the maxThreads parameter in libavif. For more details, see the
-   *     documentation for maxThreads variable in avif.h.
+   * @param threads Number of threads to be used for the AVIF decode. Zero means use the library
+   *     determined optimal value as the thread count. Negative values mean use number of CPU cores
+   *     as the thread count. When this value is > 0, it is simply mapped to the maxThreads
```

Comment:
The sentence "When this value is > 0, it is simply mapped to the maxThreads parameter in libavif." is a little confusing. It seems to imply that the maxThreads parameter in libavif is only set when the value of the `threads` parameter is > 0.

Reply:
Removed the sentence about > 0.